io/pci/dlpar.py

```diff
@@ -88,6 +88,7 @@ def setUp(self):
         if not self.server:
             self.cancel("Managed System not got")
         self.lpar_2 = self.params.get("lpar_2", '*', default=None)
+        self.lpar2_id = self.params.get("lpar2_id", '*', default=None)
```

Comment:
Why are we getting this from user, we can find this via commands.
https://github.com/avocado-framework-tests/avocado-misc-tests/blob/dbe5d1eda224d06989c9d67959eb8749ff3e5bce/io/pci/dlpar.py#L105

Reply:
ok